<?xml version="1.0"?>
<launch>

    <node pkg="hrl_thermal_camera" type="test_converter" name="image_converter" output="screen"/>

    <!--node ns="thermal_camera_driver" pkg="image_proc" type="image_proc" name="thermal_camera_image_proc" output="screen">
    </node-->

    <node name="thermal_camera" pkg="hrl_thermal_camera" type="hrl_thermal_camera" output="screen">
      <rosparam command="load" file="$(find hrl_thermal_camera)/params/thermal_camera_params.yml"/>
      <param name="framerate" value="10"/>
    </node>




</launch>
